<?xml version="1.0" encoding="UTF-8"?>
<launch>


  <!-- COLLISION GROUND PARAM -->
  <arg name="collision_ground"   default="true"/>
  <arg name="world_name" default="box.world"/>

  <node pkg="timed_roslaunch" type="timed_roslaunch.sh" args="0 icclab_grasping_niryo base_world_model.launch world_name:=$(arg world_name)" name="timed_base_world_model" />
	  
  <node pkg="timed_roslaunch" type="timed_roslaunch.sh" args="5 icclab_grasping_niryo controllers.launch" name="timed_controllers" />

  <node pkg="timed_roslaunch" type="timed_roslaunch.sh" args="10 niryo_one_bringup robot_commander.launch" name="timed_commander" />

  <node pkg="timed_roslaunch" type="timed_roslaunch.sh" args="15 icclab_grasping_niryo rviz_grasps_filter_sim.launch" name="timed_filter" />

  <node pkg="icclab_grasping_niryo" type="control_gripper.py" args="--sim" name="control_gripper"/> 

</launch>
